ktor-server/ktor-server-core/jvm/src/io/ktor/server/http/content/StaticContent.kt

```diff
@@ -249,14 +252,61 @@ public fun Route.staticZip(
     basePath: String?,
     zip: Path,
     index: String? = "index.html",
+    reloadOnRequest: Boolean = false,
```

Comment:
Maybe we can avoid introducing this parameter and instead always use the auto-reloading?  I don't think it will have any significant drawbacks and should make things a lot easier.